### DISCUSSION TITLE
Stuck on "Generating SSL certificate" for domain

### DISCUSSION THREAD

**Discussion by ateethk** [vercel/community#3289] [Help] [answered]
### Summary

Hi guys,

I've been getting "Generating SSL certificate" for the past day on a domain that I'm pointing to Vercel from Porkbun. I have A, CAA, and CNAME records set but still unable to generate SSL. I was actually able to generate SSL through Porkbun, but Vercel won't let me upload the cert. 

Would anyone please be able to help me figure out what I'm doing wrong? The domain is www.stonemarket.io.

Thanks,
Ateeth

### Example

https://www.stonemarket.io or www.stonemarket.io

### Steps to Reproduce

Current records are set as"

A | stonemarket.io | 76.76.21.21 | 600
CAA | stonemarket.io | 0 issue "letsencrypt.org" | 600
CNAME | www.stonemarket.io | cname.vercel-dns.com. | 600

I have DNSSEC disabled per my knowledge too.

**Comment by swarnava** [comment, marked as answer]
Hey, 

It looks like the DNS isn't pointing to Vercel anymore.
